lib/game.rb

```diff
@@ -0,0 +1,131 @@
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+
+class Game
+    def initialize    
```

Comment:
If your `initialize` method is going to be empty, you do not need to define it at all. A class will work without it being explicitly defined.